lib/flash_runner.rb

```diff
@@ -0,0 +1,44 @@
+require './lib/card'
+require './lib/turn'
+require './lib/deck'
+require './lib/round'
+
+
+@card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+@card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+@card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+@cards = [@card_1, @card_2, @card_3]
+@deck = Deck.new(@cards)
+@round = Round.new(@deck)
+
+def start
+
+  puts "Welcome! You're playing with #{@round.deck.cards.count} cards."
+  puts "--------------------------------------------------------------"
+  loop_cards
+end
+
+
+def loop_cards
```

Comment:
This is a good method! I like how dynamic it is and that it will change on its own depending on how many cards are in a deck.